src/connectionconfig/connectionDialogWebviewController.ts

```diff
@@ -26,7 +26,10 @@ import {
 import { ConnectionOption } from "azdata";
 import { Logger } from "../models/logger";
 import VscodeWrapper from "../controllers/vscodeWrapper";
-import { ConnectionDialog as Loc, refreshTokenLabel } from "../constants/locConstants";
+import {
```

Comment:
Are there any code changes in these files, or is it just formatting?

Reply:
Just prettier formatting.